### DISCUSSION TITLE
nestjs-open-telemetry

### DISCUSSION THREAD

**Discussion by giscus** [tomwray13/tomray.dev#69] [Blog Comments]
# nestjs-open-telemetry

A deep dive tutorial into how to use Open Telemetry in NestJS applications to instrument, generate, collect and export telemetry data.

https://www.tomray.dev/nestjs-open-telemetry

**Comment by Thrillz** [comment]
Great article and really easy to follow. Quick question, if i want to set up grafana-tempo to consume the jaeger traces, what will the (endpoint?) setting for my jaeger receiver be. I currently have it as localhost:14268/api/traces like in the article but grafana tempo does not pick up the jaeger traces.
